reflex/utils/prerequisites.py

```diff
@@ -625,7 +612,7 @@ def _update_next_config(
     next_config = {
         "basePath": config.frontend_path or "",
         "compress": config.next_compression,
-        "reactStrictMode": True,
+        "reactStrictMode": config.react_strict_mode,
```

Comment:
Should we just expose a `NextConfig` object? We can add all the params there eventually. And let them do `config.next_config.react_strict_mode`

Reply:
I guess we could.